src/Microsoft.Azure.Functions.Extensions.Mcp/McpToolExtensions.cs

```diff
@@ -74,4 +74,28 @@ internal static JsonElement GetPropertiesInputSchema(this IMcpTool tool)
         using var doc = JsonDocument.Parse(ms);
         return doc.RootElement.Clone();
     }
+
+    private static void WriteTypeAndEnum(Utf8JsonWriter writer, string propertyType, IReadOnlyList<string> enumValues)
+    {
+        // Write type and enum at the appropriate level
```

Comment:
nit: the method doesn't perform any validation regarding level. We can remove the comment.